src/types/enhancers.ts

```diff
@@ -141,10 +141,10 @@ type CssProps = Pick<
 type BoxCssProps<CP> = {
   // Enhance the CSS props with the ui-box supported values.
   // `string` isn't added because it'll ruin props with string literal types (e.g textAlign)
-  [P in keyof CP]: CP[P] | number | boolean | null | undefined
+  [P in keyof CP]: CP[P] | number | false | null | undefined
```

Comment:
We don't accept `true | false` – just `false`. In non-production mode `true` will spit out an error: 
https://github.com/segmentio/ui-box/blob/502f66dc68f0a72e62b74495dda8c8b67f83ee55/src/get-css.ts#L14-L15